contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
+* Multiple business units need a software component.
+* It makes sense for the business units to collaborate; each needs some different changes to the code.
+* One BU doesn't have the resources to do the development for all the rest.
+* InnerSource programs and tools exist.
+
+## Problem  
+The company needs to develop a software product component; multiple business units use this component.
+
+## Forces  
+* The product's importance to company revenue and the committed feature content and dates require a development paradigm that provides known, stable development resources (headcount).
+
+## Sketch (optional)
+
+## Solution
+The solution is for the participating BUs to each dedicate development resources to collaboratively add the features that each BU needs to the software component. InnerSource tools and some InnerSource processes are used, but the InnerSource cooperation is contained; there is no attempt to encourage open contribution from participants outside the core group of BUs that have entered into this arrangement.
```

Comment:
Is open contribution then irrelevant to this particular pattern? I.e., it doesn't matter whether the owning BU encourages it or discourages or forbids it.